platforms/platform-core/src/CoreRequest.ts

```diff
@@ -18,8 +25,12 @@ export class CoreRequest extends JovoRequest {
     return this.locale;
   }
 
+  getInput(): JovoInput {
+    return new JovoInputBuilder(super.getInput()).set('nlu', this.input?.nlu).build();
```

Comment:
Could you explain a bit further what this is doing here? Thanks!

Reply:
Thank you. I'm still not sure if I get it, to be honest. A bit confused why specifically `nlu` needs to be set in this method and why we're not setting any other property. What is a use case here?

Could the reasoning be commented or documented somewhere? i think otherwise it could happen that we don't know why this is there if we look at the code in a few months

Reply:
Okay so I guess this could be seen as being redundant, the original idea was to access the nlu properties in the handler, but since intents and entities are now accessed in `getIntent()` and `getEntities()` if not set in the root properties this can be removed.

Reply:
@rubenaeg How do we access other properties that we might add to $input.nlu?
This could be an output object with messages if the SLU/NLU provides dialog management or messages.

Reply:
Well I now removed the code, but you can see in the code above how you could add properties to the input object that is returned by `getInput()`. @jankoenig do you need more input on this PR? I can write some documentation if you'd like, something like "Extending the Input object", where I could also cover ambient declarations...